buildmaster.sh

```diff
@@ -291,14 +292,11 @@ Summary of chosen actions:
  Do Git pull on src  [$(get_option $OPT_GIT_PULL)]
  Use alt. branch ... [$OPT_BRANCH]
  SDK Config Variant  [$OPT_VARIANT]
+ SDK Release Name .. [$OPT_VARIANT_PRETTY]
```

Comment:
It might be misleading that it should change with every release, while we know it is desired to remain stable. So maybe "SDK Pretty Variant" would align nicely.

Reply:
Updated